Update upgrade (#8985)
fishtown-analytics/docs.getdbt.com: website/docs/docs/fusion/fusion-readiness.md

The <Constant name="fusion_engine" /> is here! We currently offer it as a [private preview](/docs/dbt-versions/product-lifecycles#the-dbt-platform) on the <Constant name="dbt_platform" />. Even if we haven't enabled it for your account, you can still start preparing your projects for upgrade. Use this checklist to ensure a smooth upgrade once <Constant name="fusion" /> becomes available. If this is all new to you, first [learn about <Constant name="fusion" />](/docs/fusion), its current state, and the features available. 

:::note Enable the Fusion readiness panel

> The <Constant name="fusion" /> readiness panel in the <Constant name="dbt_platform" />, which shows each project's eligibility status and blockers, is being rolled out in phases. If it hasn't been automatically turned on for your account yet, an account admin can manually enable the **Enable Fusion readiness & upgrade features** in **Account settings** → **Account**. See [Enable Fusion readiness features](/guides/prepare-fusion-upgrade?step=2) for setup steps.

Reviewer: ```suggestion
The <Constant name="fusion" /> readiness panel in the <Constant name="dbt_platform" /> and shows each project's eligibility status and blockers, is being rolled out in phases. If it hasn't been automatically enabled for your account yet, an [account admin](/docs/cloud/manage-access/enterprise-permissions#account-admin) can manually enable the **Enable Fusion readiness & upgrade features** in **Account settings** → **Account**. Refer to [Enable Fusion readiness features](/guides/prepare-fusion-upgrade?step=2) for setup steps.
```

> The Fusion readiness panel in the <Constant name="dbt_platform" />, which shows each project's eligibility status and blockers, is only visible after an account admin enables **Enable Fusion readiness & upgrade features** in **Account settings** → **Account**. You can work through this checklist at any time, but you'll need that setting enabled to track your readiness status in the UI. See [Enable Fusion readiness features](/guides/prepare-fusion-upgrade?step=2) for setup steps.

Reviewer 2: This is where it gets a little tricky because we will do a cohorted roll-out (starting with 10% of snowflake on monday). If they're in the roll-out, we will automatically enable the readiness experience to their account. The remaining 90% of snowflake accounts and other adapter accounts will need to turn on this setting to see it.

That will scale over time as we scale the roll-out

> The Fusion readiness panel in the <Constant name="dbt_platform" /> — which shows each project's eligibility status and blockers — is only visible after an account admin enables **Enable Fusion readiness & upgrade features** in **Account settings** → **Account**. You can work through this checklist at any time, but you'll need that setting enabled to track your readiness status in the UI. See [Enable Fusion readiness features](/guides/prepare-fusion-upgrade?step=2) for setup steps.

Author: ```suggestion
The Fusion readiness panel in the <Constant name="dbt_platform" />, which shows each project's eligibility status and blockers, is only visible after an account admin enables **Enable Fusion readiness & upgrade features** in **Account settings** → **Account**. You can work through this checklist at any time, but you'll need that setting enabled to track your readiness status in the UI. See [Enable Fusion readiness features](/guides/prepare-fusion-upgrade?step=2) for setup steps.
```
